lib/api_web/controllers/error_controller.ex

```diff
@@ -1,48 +1,53 @@
-defmodule ApiWeb.Controller.Errors do
+defmodule ApiWeb.ErrorController do
   use Api.Web, :controller
 
   alias ApiWeb.{ChangesetView, ErrorView}
 
-  def changeset(conn, data) do
+  def call(conn, {:error, changeset}) do
```

Comment:
Let's force the type on matching. Something like `{:error, %Ecto.Changeset{} = changeset}`.